app/pages/collections/list.cjsx

```diff
@@ -65,6 +71,10 @@ List = React.createClass
       .then (collections) =>
         @setState {collections}
 
+  shared: (collection) ->
+    if (@props.params.collection_owner is @props.user.login) or (@props.params.profile_name is @props.user.login)
```

Comment:
It's not breaking the app, but there is a TypeError being thrown if you load the `collections/:collection_name` route directly while it's waiting for the user authentication. Adding the existential operator to `@props.user` in the conditional should fix it.